components/central-application-gateway/README.md

```diff
@@ -0,0 +1,100 @@
+# Application Gateway
+
+## Overview
+
+This is the repository for the Kyma Application Gateway.
+
+## Prerequisites
+
+The Application Gateway requires Go 1.8 or higher.
+
+## Installation
+
+To install the Application Gateway, follow these steps:
+
+1. `git clone git@github.com:kyma-project/kyma.git`
+2. `cd kyma/components/application-gateway`
+3. `CGO_ENABLED=0 go build ./cmd/applicationgateway`
+
+## Usage
+
+This section explains how to use the Application Gateway.
+
+### Start the Application Gateway
+
+To start the Application Gateway, run this command:
+
+```
+./applicationgateway
+```
+
```

Comment:
The documentation should describe HTTP API / endpoints for Legacy and compass modes.